R/data_export.R

```diff
@@ -21,10 +24,11 @@
 #' @examples
 #' \dontrun{
 #' export_data(COW, database = "states",
-#' URL = "https://correlatesofwar.org/data-sets/state-system-membership")
+#' URL = "https://correlatesofwar.org/data-sets/state-system-membership",
+#' package = "qStates")
 #' }
 #' @export
-export_data <- function(..., database, URL) {
+export_data <- function(..., database, URL, package = NULL) {
```

Comment:
Can't we infer the package name by the time users `export_data()`? Maybe write a `get_package_name()` helper that can be used elsewhere too?